mobile/src/navigations/onboarding-navigation/components/step/step.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+
+import { Pressable, Text, View } from '~/bundles/common/components/components';
+import { Color, IconName, TextCategory } from '~/bundles/common/enums/enums';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+type Properties = {
+    stepNumber: number;
+    isFocused: boolean;
+    isCompleted: boolean;
+    disabled: boolean;
+    routeName: string;
+    onPress: () => void;
+};
+
+const Step: React.FC<Properties> = ({
+    stepNumber,
+    onPress,
+    isFocused,
+    isCompleted,
+    disabled,
+    routeName,
+}) => {
+    const number = 1;
+    // prettier-ignore
+    return (
+        <Pressable
+            key={stepNumber}
+            onPress={onPress}
+            disabled={disabled}
+            style={[
+                globalStyles.flexDirectionRow,
+                globalStyles.alignItemsCenter,
+                globalStyles.m25,
+                styles.singleStep,
+            ]}
+        >
+            {isCompleted ? (
+                <Icon
+                    name={IconName.CHECK_CIRCLE}
+                    color={Color.PRIMARY}
+                    size={30}
+                />
+            ) : (isFocused ? (
+                <Icon
+                    name={IconName.CIRCLE_OUTLINE}
+                    color={Color.PRIMARY}
+                    size={30}
+                    style={styles.activeIcon}
+                />
+            ) : (
+                <Icon name={IconName.CIRCLE} color={Color.INPUT} size={30} />
+            ))}
```

Comment:
If that's the reason for prettier ignore let's move this to use memo and add switch case, also let's replace isCompleted, isFocused and disabled with some enum (StepState for ex.)

Reply:
I should just create an object in the Steps component and pass to the Step? Because Im a bit confused. 
Also I can't use switch case because of `Type "boolean" is not comparable` so I guess that I should use If else instead.